pkg/daemon/daemon.go

```diff
@@ -542,6 +547,15 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 			if e := mkFifo(); e != nil {
 				glog.Errorf("Error creating named pipe, GNSS monitoring will not work as expected %s", e.Error())
 			}
+
+			for _, section := range output.sections {
+				if section.sectionName == "[global]" {
```

Comment:
There will be always leafile field in config since customer using reference config. In that case you should move this as part of ln 492 above
if section.sectionName == "[global]" {

Reply:
We removed the "leapfile" key from the reference config. Also, moving it up to 492 will result in the "leapfile" key appearing in the global section of every config file, not just ts2phc config...